dbg.h

```diff
@@ -71,7 +71,7 @@ License (MIT):
 
 namespace dbg {
 
-#ifdef DBG_MACRO_UNIX
+#if defined(DBG_MACRO_UNIX) && !defined(DBG_MACRO_FORCE_COLOR)
```

Comment:
This implementation is a bit confusing, as it intertwines the Unix/non-Unix logic with the "force color" logic. I'd rather see a version that changes the implementation of the Unix version:
```c++
inline bool isColorizedOutputEnabled() {
  return defined(DBG_MACRO_FORCE_COLOR) || isatty(fileno(stderr));
}
```

The compiler should still optimize this to `return true`.

Reply:
That particular example doesn't compile since defined(...) seems to only work with preprocessor directives

Reply:
Oh, of course. Thank you for the update!